Template10 (Library)/Controls/HamburgerMenu.PublicProperties.xaml.cs

```diff
@@ -334,6 +334,7 @@ public SplitViewDisplayMode VisualStateNarrowDisplayMode
             DependencyProperty.Register(nameof(VisualStateNarrowDisplayMode), typeof(SplitViewDisplayMode),
                 typeof(HamburgerMenu), new PropertyMetadata(SplitViewDisplayMode.Overlay, (d, e) =>
                 {
+                    (d as HamburgerMenu).UpdateVisualStates();
```

Comment:
Let's change this so there is no logic in this file. Use the InternalChanged methods to add functionality in the Ham.xaml.cs file. Okay?